### PR TITLE
50 user sees message chain

### PR DESCRIPTION
**Pull request by rvna**
This allows a requester or professional to see a conversation in reverse chronological order. If it looks good on review, it is ready for development.

### CODE REVIEW

File: app/controllers/messages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class MessagesController < ApplicationController
+  def index
+    @messages = current_user.messages
+                            .where(job_id: params[:job])
+                            .where('sender_id = ? OR recipient_id = ?', params[:with], params[:with])
+                            .order(created_at: :desc)
```
Comment (JeanJoeris) [inline app/controllers/messages_controller.rb]: I feel like we might want to refactor this. maybe something like 

```
current_user.messages.job_conversation(job_conversation_params)
```

where the job conversation params are the job_id and with_id and #job_conversation puts the db call into the model

**Review by JeanJoeris — APPROVED** [review]
I think we should refactor a little, but great job

File: app/controllers/messages_controller.rb (hunk 1 above)
Comment (rvna) [reply app/controllers/messages_controller.rb]: That seems like a good refactor, I updated and pushed that up.

**Comment by JeanJoeris** [comment]
Thanks for making these views

**Pull request merged**